TheJumpCloud/support
# update AppData path for pwm install

## Issues
* [SA-2475](https://jumpcloud.atlassian.net/browse/SA-2475) - Update AppData path for pwm installation script

## What does this solve?

The Application Data path for the PWM app has changed, the while loop in the pwm installation script needs to be updated to address this change from `C:\Users\someUsername\AppData\Local\pwm\JumpCloud Password Manager.exe"` to `C:\Users\someUsername\AppData\Local\jcpwm\JumpCloud Password Manager.exe"`

## Is there anything particularly tricky?

No

## How should this be tested?

On a Windows system:

- Associate a JumpCloud user to a device
- Create a new command and copy the contents from this command PR into the body
- Run the command on the device while the JumpCloud user is signed into the device
- The command should run on the device and install the JumpCloud Password Manager

## Screenshots

System running the installation script should show this logo for several seconds:
![Screen Shot 2023-08-03 at 8 36 12 AM](https://github.com/TheJumpCloud/support/assets/54448601/3d119f97-658a-4d07-acc1-688861af0647)


[SA-2475]: https://jumpcloud.atlassian.net/browse/SA-2475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
